Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -842,22 +842,45 @@ protected override void ProcessRecord()
     ///   <para>Update the contents of the Storage Object with the string "OK".</para>
     ///   <para><code>PS C:\> "OK" | Write-GcsObject -Bucket "widget-co-logs" -ObjectName "status.txt"</code></para>
     /// </example>
+    /// <example>
```

Comment:
Please rewrite the examples as follows. Now that I'm working on the cmdlet reference website, we need to be more particular about how we format the examples.

In short: no `<para>` before `<code>`. The format should be:
````
<code>single-line-code-snippet</code>
<para>description 1</para>
<para>description 2</para>

or

<code>
code line 1
code line 2
</code>
<para>super really long paragraph of text
that continues on two lines</para>
````

````
/// <example>
///   <code>PS C:\> "OK" | Write-GcsObject -Bucket "widget-co-logs" -ObjectName "status.txt"</code>
///   <para>Update the contents of the Storage Object with the string "OK".</para>
/// </example>
```